src/backend/metal/src/lib.rs

```diff
@@ -421,6 +421,14 @@ impl Instance {
             layer
         };
 
+
+        extern "C" {
```

Comment:
let's move this to the top level, perhaps? it's good to see all extern declarations in one place